packages/swap/src/utils/getPoolQuote.ts

```diff
@@ -25,7 +25,7 @@ const getPrice = (
 export default async function getPoolQuote({
   srcAsset,
   destAsset,
-  swapInputAmount: originalSwapInputAmount,
```

Comment:
i think we initially deducted the ingress fee outside this method. but then with `swap_rate_v2` the ingress fee calculation moved into the rpc and therefore the value we pass here is the deposit amount